### PR TITLE
Added repairshopr ticketing system

### PR DESCRIPTION
**Pull request by thegeekkid**

### CODE REVIEW

**Review by stephengroat — APPROVED** [review]

File: _data/other.yml [hunk 1]
```diff
@@ -279,6 +279,16 @@ websites:
       img: promoter.png
       tfa: No
       twitter: Promoter_io
+      
+    - name: Repairshopr
+      url: http://www.repairshopr.com/
```
Comment (Jawshy) [inline _data/other.yml]: HTTPS also works which 2factorauth prefers.

Comment (thegeekkid) [reply _data/other.yml]: HTTPS only redirects to HTTP on their main site.  All login pages and the actual product use https; however, their homepage and documentation all uses HTTP.  :/

Comment (thegeekkid) [inline _data/other.yml]: Sorry, all login pages and the actual product use *https* but everything else uses http.

Comment (Carlgo11) [reply _data/other.yml]: Are you sure? It doesn't redirect for me.
screenshot

Comment (thegeekkid) [reply _data/other.yml]: Ahhh - I figured it out.  When you go to https://www.repairshopr.com it works; but I was just going to https://repairshopr.com; it redirects to http://repairshopr.com.  Poor implementation on their part; but I'll update it as soon as I get on a real computer (and not my phone).  :)

Comment (Carlgo11) [reply _data/other.yml]: It's the same for the doc link btw 😉

Comment (thegeekkid) [reply _data/other.yml]: Yep.  Now I feel really stupid - lol!  Just updated both of them.  :)

**Pull request merged**